react/features/base/participants/middleware.js

```diff
@@ -407,6 +419,26 @@ function _registerSounds({ dispatch }) {
     dispatch(registerSound(PARTICIPANT_LEFT_SOUND_ID, PARTICIPANT_LEFT_FILE));
 }
 
+/**
+ * Parses the userInfo from the URL and updates the local participant if needed.
+ *
+ * @param {Store} store - The redux store.
+ * @private
+ * @returns {void}
+ */
+function _updateLocalParticipantFromUrl({ dispatch, getState }) {
```

Comment:
I'm not sure about this change. base/settings takes care of user settings, and then it updates the local user with those. For example, when you configure the email in the settings menu. Now, you moved half of that elsewhere, so one needs to go to here to check that. It doesn't sound good.

The same way the config is overridden from the URL, some settings are (ie, this), that's why the code was where it was.

Reply:
ok. I still think it doesn't belong here because it has nothing to do with the local settings but your explanation also makes sense. anyway I don't have strong preference about this and i'll revert the change...